features/built_in_matchers/include.feature

```diff
@@ -11,6 +12,9 @@ Feature: `include` matcher
 
     expect([1, 2]).to include(1)
     expect([1, 2]).to include(1, 2)
+    expect([1, 2]).to include(be_an(Integer))
```

Comment:
This reads oddly and given the examples below is a bit superfluous, maybe drop it?